test/inductor/test_multi_kernel.py

```diff
@@ -91,6 +97,56 @@ def test_softmax(self, expect_multi_kernel=True):
         else:
             self.assertFalse(_contains_multi_kernel_code(wrapper_code))
 
+    @requires_triton()
+    @unittest.skipIf(not IS_BIG_GPU, "templates require big gpu")
+    def test_triton_gemm(self):
+        def fn(x, y):
+            return x @ y
+
+        compiled_fn = torch.compile(
+            fn,
+            options={
+                "max_autotune": True,
+                "max_autotune_gemm_backends": "TRITON",
+            },
+        )
+        x = torch.randn(4096, 4096, device="cuda")
```

Comment:
Hi, can you repalce the `"cuda"` with `GPU_TYPE` in this case to avoid `no cuda` crash when runing on other GPU.